examples/secrets/latest.py

```diff
@@ -0,0 +1,11 @@
+# [START import]
+from nitric.api import Secrets
+# [END import]
+async def secret_latest():
+# [START snippet]
+    # Construct a new secret client
+    secrets = Secrets()
+
+    # Get latest secret version
+    value = secrets.secret("my-secret").latest()
```

Comment:
This variable should be called `version`, since latest returns a ref to the latest version, not the secret value. Alternatively, the line could be changed to:

```python
value = await secrets.secret("my-secret").latest().access()
```